modules/primer-pagination/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "version": "0.0.1",
```

Comment:
Will probably want to start the versioning at `1.0.0`?

Reply:
ok!